batchflow/models/torch/blocks/named_blocks.py

```diff
@@ -408,3 +408,54 @@ def __init__(self, inputs=None, layout='Rwncac!', channels='same', expand_ratio=
             **kwargs
         }
         super().__init__(inputs=inputs, layout=layout, **kwargs)
+
+
+class MSCANBlock(Block):
+    """ MultiScaleConv block: modern Inception-like block with some adaptation from Transformers.
+    Meng-Hao Guo et al. "`SegNeXt: Rethinking convolutional attention design for Semantic Segmentation
+    <https://arxiv.org/abs/2209.08575v1>`_"
+
+    In the default case, the layout is:
+        Rnca Rc Rm+ c* c+ Rnccac+
+        │    │  └0┘  │  │ └──3──┘
+        │    └───1───┘  │
+        └───────2───────┘
+        - `0` is multi scale conv: multiple factorized kernels, combined into one tensor.
+        - `1` uses multi-scale conv as re-weighting for input tensor.
+        - `2` adds normalization and activation layers.
+        - `3` stacks an MLP block, inspired by recent transformers.
+
+    Parameters
+    ----------
+    msca_kernel_size : sequence of ints
+        Kernel sizes in multi-scale convolution.
+    add_mlp : bool
+        Whether to stack an additional MLP block on top.
+    mlp_expansion : int
+        Expansion ratio for channels in MLP block.
+    """
+    def __init__(self, inputs=None, layout='Rnca (Rc Rm! c*) c!', msca_kernel_size=(7, 11, 15),
```

Comment:
```suggestion
    def __init__(self, inputs=None, layout='Rnca (Rc Rm! c*) c!', kernel_size=(7, 11, 15),
```

Reply:
This would conflict with `c` layer: these are specific for multi-scale convolutions only